packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -15,16 +22,16 @@ Source0:    https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?prerel
 BuildArch:  noarch
 
 Requires:   curl
-Requires:   %{?scl_prefix}puppet-agent >= 5.5.10
+Requires:   puppet-agent >= 5.5.10
 Requires:   %{?scl_prefix}rubygem(kafo) >= 4.0.0
 Requires:   %{?scl_prefix}rubygem(kafo) < 5.0.0
 Requires:   foreman-selinux
 Requires:   %{?scl_prefix_ruby}ruby(release)
 Requires:   foreman-maintain
 
 BuildRequires: asciidoc
-BuildRequires: rubygem(rake)
-BuildRequires: %{?scl_prefix}puppet-agent >= 1.9.0
+BuildRequires: puppet-agent >= 1.9.0
```

Comment:
1.9.0 was Puppet 4. We didn't bump it then because the build root still had Puppet 4, but it's probably safe to require `>= 5.5.0` now. Perhaps also define it as a macro so they don't go out of sync?